engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -23,6 +20,8 @@ import scala.concurrent.{Await, Promise}
 import scala.language.postfixOps
 
 object WorkflowManagerActor {
+  val DefaultMaxWorkflowsToLaunch = 50
+  val DefaultNewWorkflowPollRate = 20
```

Comment:
I disagree. Having it as a pull means that we can tune the WMA to accept more/less work for different situations. Granted there's a lot more complexity than "number of workflows in flight" but the reality is that people are already using that number for the applications which they're building external to Cromwell.

Also I don't see why "WMA doesn't have to change" is a big deal, it's a minor change and IMO a bit cleaner anyways.